### ISSUE TITLE
Shipper HQ autocomplete address Download failed

### ISSUE THREAD

**Issue by nrauf90** [shipperhq/module-address-autocomplete#23]
I'm trying to update my Magento2 version and PHP version of my local environment. For this when I'm trying to run the composer update or install command it shows an error on the address autocomplete module. All other Shipper HQ module downloaded 100%.

 [Composer\Downloader\TransportException]
  The "https://repo.magento.com/archives/shipperhq/module-address-autocomplete/shipperhq-module-address-autocomplete-
  20.3.0.0.zip" file could not be downloaded (HTTP/1.1 404 Not Found)

**Comment by ibraheemnabeelfauzi** [comment]
Hey @nrauf90  can you please tell me if you followed the install/update instructions in here? https://github.com/shipperhq/module-address-autocomplete 

If so then please paste me a screenshot from your CLI with the error and the commands you're running.  

Thanks.

**Comment by nrauf90** [comment]
Yes, we followed all the instruction that mentioned. It was working few days before where I  update everything on our live site. For the last two days, I'm trying to update my local development system where I get this msg on the composer install command.

**Comment by ibraheemnabeelfauzi** [comment]
@nrauf90  what version of composer are you running? Can you share with me the steps you took or a screenshot from the CLI if possible - could be a composer issue here?

**Comment by wsajosh** [comment]
Closing this issue as we didn't receive a response. Please re-open if you're still having issues
